<ui version="4.0" >
 <class>ProfileWidgetUI</class>
 <widget class="QWidget" name="ProfileWidgetUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>388</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QListView" name="listView" />
   </item>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" name="layout" >
     <item>
      <widget class="KPushButton" name="kpushbutton_add" >
       <property name="text" >
        <string>Add...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="kpushbutton_rem" >
       <property name="text" >
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="kpushbutton_mod" >
       <property name="text" >
        <string>Modify...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="kpushbutton_copy" >
       <property name="text" >
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>32</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="kpushbutton_init" >
       <property name="text" >
        <string>Codec Scan</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>93</width>
         <height>131</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="kpushbutton_load" >
       <property name="text" >
        <string>Load...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="kpushbutton_save" >
       <property name="text" >
        <string>Save...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
